src/content/en/2022/privacy.md

```diff
@@ -1,22 +1,466 @@
+  content="82%",
+  classes="big-number",
+  sheets_gid="225736044",
+  sql_file="most_common_tracker_categories.sql"
+)
+}}
+
+Tracking is one of the most pervasive web technologies on the web - we find that 82% of desktop websites (80% for mobile) include at least one third-party tracker. By following users' behavior online, these tracking companies can create profiles of them, which can be used for personalized advertising, give insights to website owners on who visits their websites, or use this information to distinguish legitimate users from (unwanted) bots. In this section we explore the different techniques that are used to track the activities of users online and look at how trackers aim to circumvent the various privacy features that aim to protect users from being tracked.
+
+### Third-party tracking
+
+One of the most common forms of online tracking is through third-party services, where a website owner typically includes a third-party (cross-site) script that provides site analytics or shows advertisements to the visitors. This script can then set a third-party cookie, and register which website the user visited. Whenever the user visits another website that includes the same third-party service, the cookie will be sent along to the tracker, allowing them to re-identify the user and link both website visits to the same profile.
+
+The types of third-party services that are included, and by doing so are implicitly given the capabilities of tracking website visitors, somewhat vary. The two most common categories ([as defined by WhoTracks.me](https://whotracks.me/blog/tracker_categories.html)) of such trackers are site analytics scripts (68% on mobile, 73% on desktop) and advertising (66% on mobile, 68% on desktop). These two are followed by a couple other categories, some of which might not have a clear link to tracking: customer interaction (services that allow customers to easily send messages to the website owner), audio/video players (e.g. YouTube embedded videos), and social (e.g. Facebook like buttons).
+
+{{ figure_markup(
+  image="most-common-trackers.png",
+  caption="Most common trackers.",
+  description="Bar chart showing the prevalence of trackers on web pages. Google Analytics (site analytics) can be found on 65% desktop sites and 60% of mobile sites, Google (advertising) on 51% and 49% respectively, DoubleClick (advertising) on 50% and 46%, Facebook (advertising) on 30% and 28%, Google Adservices (advertising) on 23% and 21%, Google Syndication (advertising) on 12%, WordPress Stats (site analytics) on 6% for both mobile and desktop, Twitter (Social Media) on 6% and 5%, Adobe Audience Manager (advertising) on 5% and 6%, and finally Rubicon (advertising) on 5% for both mobile and desktop sites.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRSnqWDZQ07x6NdnIix7JOfg5DqGcoTrrpwaKavCIvNFkIANncyMxahKaTYjLka3rsrcBmrRIURpnPo/pubchart?oid=951980374&format=interactive",
```

Comment:
This should have a height attribute set, the figure is getting cut off: https://20220926t211726-dot-webalmanac.uk.r.appspot.com/en/2022/privacy#fig-2

<img width="743" alt="image" src="https://user-images.githubusercontent.com/1120896/192383476-5de675e2-97c0-4c51-8321-f5951620fa72.png">

You can get the exact height by going back to the publishing info in Sheets:

<img width="982" alt="image" src="https://user-images.githubusercontent.com/1120896/192383616-e9436942-72d2-4253-86c8-b844cbb3a7fd.png">

Reply:
Added height attribute for all graphs with non-default height in a39b587705a3d90cbd9ed62ef1c821e162dcaaec